h1st/model/fuzzy/fuzzy_logic_modeler.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from typing import List, Union
+
+import skfuzzy
+from skfuzzy import control as skctrl
+
+from h1st.model.modeler import Modeler
+from h1st.model.fuzzy.fuzzy_logic_model import FuzzyLogicModel
+from h1st.model.fuzzy.fuzzy_logic_rules import FuzzyLogicRules
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class FuzzyLogicModeler(Modeler):
+    """
+    Base class for H1st Fuzzy Logic Modelers. Has capabilities that are 
+    specific to FuzzyLogicModel.
+
+    """
+    def __init__(self, model_class = FuzzyLogicModel):
+        self.model_class = model_class
+
+    def build_skfuzzy_control_system(self, fuzzy_rules: FuzzyLogicRules):
```

Comment:
Since this is quite simple, it is actually easier to read if it is *not* separated into its own method.

Related, you should shorten variable names when the context is clear. For example, there is no need to use `fuzzy_logic_rules` when `rules` will do very well, given the context of the object/method being invoked.